airflow/settings.py

```diff
@@ -148,6 +148,11 @@ def configure_orm(disable_connection_pool=False):
     Session = scoped_session(
         sessionmaker(autocommit=False, autoflush=False, bind=engine))
 
+
+def configure_action_logging():
```

Comment:
maybe a docstring for the function?

Reply:
Done

Reply:
Are cluster administrators expected to override this? And if so, should they do it here, or in their local_settings module?

The latter seems cleaner to me, since then they wouldn't have to deal with merge conflicts.